src/app/cardsView/base-view.component.html

```diff
@@ -0,0 +1,23 @@
+<div class="row page-header">
+  <div class="col-md-6">
+    <ol class="aui-nav aui-nav-breadcrumbs __skate">
+      <li><a href="#">Home</a></li>
+      <!--<li class="aui-nav-selected">Projects</li>-->
```

Comment:
Tu akurat może zostać. Cały element jest przenoszony to oddzielnego komponentu, nie przykładałbym do zbyt dużej wagi.